### ISSUE TITLE
User cannot change log level of Desktop application

### ISSUE THREAD

**Issue by Vere-Grey** [trezor/trezor-suite#15670]
**Describe the bug**
Desktop application ignores CLI parameter --log-level and logs at INFO level all the time.
[Docs](https://github.com/trezor/trezor-suite/blob/develop/docs/features/desktop-logger.md)

**Info:**

-   Remembered Wallet: no
-   Suite Version: app - local develop Trezor-Suite-24.12.0-linux-x86_64.AppImage
-   OS: Ubuntu 24.04.1 LTS

**How to reproduce**

1. Build locally or download AppImage
1. Start it parametrized: `./Trezor-Suite-24.12.0-linux-x86_64.AppImage --no-sandbox --log-level=error`
1. Notice WARN and INFO log entries in terminal

**Expected behavior**
Log level parameter affects what log entries are produced according to the docs

**Screenshots**
```
2024-12-02T10:55:48.847Z - INFO(build): - Dev: No
2024-12-02T10:55:48.847Z - INFO(build): - Args: --no-sandbox --log-level=error
2024-12-02T10:55:48.847Z - INFO(build): - CWD: /home/mcihlar/git/trezor-suite/packages/suite-desktop/build-electron
2024-12-02T10:55:49.004Z - INFO(modules): Initializing 25 modules
2024-12-02T10:55:49.112Z - WARN(auto-updater): Feed url: https://data.trezor.io/suite/releases/desktop/latest
```

**Comment by matejkriz** [comment]
You are right, it was introduced in https://github.com/trezor/trezor-suite/pull/14435. 

It seems to me, that it would be enough to just remove [this line `logger.level = isDevEnv ? 'debug' : 'info';`](https://github.com/trezor/trezor-suite/pull/14435/changes#diff-47d2177ee9f884e72cc2f33239984a0fa31fa14ca6e0695249ab0329f7c6cddcR84), but I don't have the full context. @martykan, could you please take a look?

**Comment by martykan** [comment]
Seems like a leftover from some debugging. 😬
Will remove.

**Comment by evgenysl** [comment]
QA NOK

I still see DEBUG and INFO when running Suite with `--log-level=error`

same for `--log-level=warn`

**Info:**
 - Suite version: desktop 26.9.0 (fef1d5a67d9fdd506ef61c5a3e3418900fed9619)
 - Browser: Trezor Suite 26.9.0
 - OS: MacIntel
 - Screen: 1920x1080
 - Device: Trezor T3W1 2.12.4 universal (revision 30be4e8c9488eeab68f994af23b3d9c9b7334266)
 - Transport: BluetoothTransport, BridgeTransport 3.2.1
